resources/phing/AtoumTask.php

```diff
@@ -171,6 +178,21 @@ public function execute()
 			}
 		}
 
+		if ($this->telemetryEnabled())
+		{
+			if (class_exists('\mageekguy\atoum\reports\telemetry') === false) {
+				throw new exception('AtoumTask depends on atoum/reports-extension being installed to enable telemetry report');
```

Comment:
The dependency is not optional if you enabled telemetry, its a mandatory.  
But if you don't enable telemetry the dependency is optional.